frontend/packages/core/src/AppProvider/registrar.tsx

```diff
@@ -10,7 +10,11 @@ const workflowRoutes = (
 ): ConfiguredRoute[] => {
   const workflowConfig = configuration?.[workflowId] || {};
   const allRoutes = Object.keys(workflowConfig).map(key => {
-    // if workflow does not contain route with user-specified key return an empty object
+    // if workflow contains an icon, return an empty object
+    if (key === "icon") {
+      return {} as ConfiguredRoute;
+      // if workflow does not contain route with user-specified key return an empty object
+    }
```

Comment:
similar q to https://github.com/lyft/clutch/pull/2677/files#r1219985997 curious why icon is handled as part of configuredroute?

Reply:
answered above